ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```
@@ -11,7 +11,7 @@ build_root:
 raw_steps:
 - pipeline_image_cache_step:
     commands: |
-      cargo build --release
+      cargo build --release && git diff --exit-code
```

```suggestion
      echo "Checking Cargo.lock"
      cargo generate-lockfile && git diff --exit-code
      echo "Building release binaries"
      cargo build --release
```

So that this would a bit more explicit?

---

I'm not sure how error handling works in this `commands` property, so I did something like your suggestion, but with `&&` chaining, with eb0197004 -> 1ae3cfc09.  That way I know we'll fail if any of the constituent commands fails.